File: 01_Basico/02 - Controle_Selecao/Resposta_em_Kotlin/questao02.kt
fun main () {
    val km_por_litro = 20
    val valor_do_litro = 5
    val qtd_dinheiro = 10
    val qtd_litros_comprada = qtd_dinheiro / valor_do_litro
    val distancia_maxima = qtd_litros_comprada * km_por_litro
    print("Pelo valor de R$ $qtd_dinheiro é possível comprar $qtd_litros_comprada L. O carro pode percorrer: $distancia_maxima km")   
}

File: 01_Basico/02 - Controle_Selecao/Respsotas_em_Python/questao02.py
km_por_litro = float(input("Km por litro: "))
valor_do_litro = float(input("Valor litro: "))
qtd_dinheiro = float(input("Valor disponivel em R$: "))
qtd_litros_comprada = qtd_dinheiro / valor_do_litro
distancia_maxima = qtd_litros_comprada * km_por_litro
print(f"Pelo valor de R$ {qtd_dinheiro} é possível comprar {qtd_litros_comprada} L. O carro pode percorrer: {distancia_maxima} km")